src/main/java/org/openstreetmap/atlas/checks/flag/FlagVisualizer.java

```diff
@@ -1,49 +0,0 @@
-package org.openstreetmap.atlas.checks.flag;
```

Comment:
Why delete?

Reply:
It isn't being used anywhere. I cleared this with @mgcuthbert , just minor housekeeping.